common/persistence/cassandra/cassandraHelpers.go

```diff
@@ -101,6 +101,7 @@ func LoadCassandraSchema(
 	defer os.Remove(tmpFile.Name())
 
 	for _, file := range fileNames {
+		// #nosec
```

Comment:
@carlcortright can you please provide the actual security violation.

@longquanzheng can you review this and provide a comment on why it is safe to suppress this security violation?

Reply:
This was flagged for a potential file inclusion vulnerability via variable. It's not an issue because we're just loading the schema. Here's the error produced by the scanner. 

```
{
  		"severity": "MEDIUM",
  		"confidence": "HIGH",
  		"rule_id": "G304",
  		"details": "Potential file inclusion via variable",
  		"file": "/home/repo/common/persistence/cassandra/cassandraHelpers.go",
  		"code": "ioutil.ReadFile(dir + \"/\" + file)",
  		"line": "104"
}
```

Reply:
That's correct. It shouldn't be an issue.